acquire/acquire.py

```diff
@@ -588,26 +588,37 @@ class Tasks(Module):
     ]
 
 
-@register_module("-nt", "--ntds")
-class NTDS(Module):
+@register_module("-ad", "--activedirectory")
```

Comment:
Historically we've explicitly kept NTDS and other AD information separate for privacy reasons (although I'm slowly becoming one of the few people to actually care about this). I'll throw it up in the team to see what the current sentiment is regarding this.

Reply:
After a discussion with my team, I would like to insist to create separate modules due to privacy reasons. I would adjust the PR.